src/Parser/Warrior/Fury/Modules/Features/Abilities.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+
+import CoreAbilities from 'Parser/Core/Modules/Abilities';
+
+class Abilities extends CoreAbilities {
+  static ABILITIES = [
+    ...CoreAbilities.ABILITIES,
+    {
+      spell: SPELLS.BLOODTHIRST,
+      category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+      getCooldown: haste => 4.5 / (1 + haste),
+      // Inner rage puts Raging Blow on a cooldown which indirectly adds more Bloodthirst to the rotation
+      // Generally 80% is recommended for Inner Rage, 30% otherwise
```

Comment:
You can toggle this by adding the spell twice and having `active` properties toggle between the one applicable to the player.